futures-sink/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "futures-sink"
+version = "0.1.0"
+authors = ["Alex Crichton <alex@alexcrichton.com>"]
+license = "MIT/Apache-2.0"
+repository = "https://github.com/alexcrichton/futures-rs"
+homepage = "https://github.com/alexcrichton/futures-rs"
+documentation = "https://docs.rs/futures-core"
```

Comment:
These documentation links will likely want to get updated, and we may want to update the repo links to the specific folders (but we can do that later of course)

Reply:
Agreed, we'll need to do a docs pass once everything is in the right place.